### PR TITLE
Fix typo in documentation

### PR DESCRIPTION
**Pull request by Ladsgroup**
Removed trailing whitespace,
Added closing bracket and used result of actual numpy.

### CODE REVIEW

File: numpy/linalg/linalg.py [hunk 1]
```diff
@@ -1023,7 +1023,7 @@ def eig(a):
     --------
     eigvals : eigenvalues of a non-symmetric array.
 
-    eigh : eigenvalues and eigenvectors of a symmetric or Hermitian 
+    eigh : eigenvalues and eigenvectors of a symmetric or Hermitian
        (conjugate symmetric) array.
```
Comment (charris) [inline numpy/linalg/linalg.py]: Should indent to match 'eigenvalues'. Not sure this adds much to 'Hermitean'.

**Comment by charris** [comment]
Commit message should begin `DOC: `. Would be good to also include
```
Removed trailing whitespace,
Added closing bracket and used result of actual numpy. 
```
in the body of the commit message. If you also include `[ci skip]` in the message body the tests won't be run.

**Comment by Ladsgroup** [comment]
All of your notes are fixed now. Thanks.

**Comment by charris** [comment]
Not quite all, but good enough ;) Thanks @Ladsgroup .

**Comment by Ladsgroup** [comment]
Thanks @charris

**Pull request merged**